packages/tag/src/components/Tag.story.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withInfo } from "@storybook/addon-info";
+import { action } from "@storybook/addon-actions";
+
+import Tag from "./Tag";
+
+storiesOf("Tag", "module")
+  .add(
+    "Default",
+    withInfo()(() => (
+      <div style={{ marginTop: "20px" }}>
+        <Tag text="THIS IS A TAG" />
+      </div>
+    ))
+  )
+  .add(
+    "Removable",
+    withInfo()(() => (
+      <div style={{ marginTop: "20px" }}>
+        <Tag text="You can remove this tag" onRemove={action("onRemove")} />
+      </div>
+    ))
+  )
```

Comment:
Create a story with value.
We have a tag with value on the `multiple tags` story. But it would be better to have documentation for this prop.